recipes/run_generators.rb

```diff
@@ -14,4 +14,6 @@
 end
 append_to_file 'lib/tasks/auto_annotate_models.rake', '# rubocop:enable Metrics/BlockLength, Rails/RakeEnvironment'
 rails_command 'generate strong_migrations:install'
+run 'rails g flipper:setup' if flipper_storage_adapter == FlipperStorageAdapters::ACTIVE_RECORD
```

Comment:
Should we run migrations here?
```suggestion
if flipper_storage_adapter == FlipperStorageAdapters::ACTIVE_RECORD
  run 'rails g flipper:setup'
  if in_docker?
    run 'rails db:migrate'
  else
    run "docker compose run --rm runner bash -c 'bundle exec rails db:migrate'"
  end
end
```

Reply:
I do not think that running migrations should be connected to the flipper setup itself. It is true that currently the only situation where we are generating any migrations is in the flipper active record, but if we run the rake db:migrate while having no migrations, it should not break e.g schema file got generated with no migrations:
<img width="1092" alt="image" src="https://github.com/user-attachments/assets/6ac89659-1d59-46fe-8546-98ccac02c912" />

So I suggest we run migrations later on, at the end of the whole process